cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -492,6 +493,16 @@ pub fn shutdown_process_compose_if_all_processes_stopped(
     Ok(all_processes_stopped)
 }
 
+/// Returns a bool indicating whether services have been started for an
+/// environment. This doesn't guarantee that the service manager is still
+/// working (e.g. hasn't crashed).
+pub fn services_probably_started(flox: &Flox, env: &dyn Environment) -> bool {
+    match env.services_socket_path(flox) {
+        Ok(socket) => socket.exists(),
+        Err(_) => false,
+    }
+}
+
```

Comment:
there may be a bit of overlap between this and #1969

Reply:
I haven't finished reviewing that PR yet but I get the impression that `ServicesEnvironment` and `guard_service_commands_available` will require a bit more work on the callers to discard errors.

Before I look further at refactoring though, what are your thoughts on whether it's worth proceeding with this PR?